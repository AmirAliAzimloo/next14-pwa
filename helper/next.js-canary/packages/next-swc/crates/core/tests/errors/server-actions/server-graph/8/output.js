/* __next_internal_action_entry_do_not_use__ {"188d5d945750dc32e2c842b93c75a65763d4a922":"$$ACTION_1"} */ import { createActionProxy } from "private-next-rsc-action-proxy";
import { encryptActionBoundArgs, decryptActionBoundArgs } from "private-next-rsc-action-encryption";
const foo = createActionProxy("188d5d945750dc32e2c842b93c75a65763d4a922", $$ACTION_1);
export async function $$ACTION_1() {
    'use strict';
}
const bar = async ()=>{
    const x = 1;
    // prettier-ignore
    'use server';
};
