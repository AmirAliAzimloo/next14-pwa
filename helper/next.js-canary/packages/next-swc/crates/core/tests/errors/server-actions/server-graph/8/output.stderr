
  x The "use server" directive must be at the top of the function body.
    ,-[input.js:9:1]
  9 |   // prettier-ignore
 10 |   'use server'
    :   ^^^^^^^^^^^^
 11 | }
    `----
